src/Components/FeelingsLog/FeelingsLog.css

```diff
@@ -0,0 +1,28 @@
+.feelingsLog {
+    --border-color: rgb(222, 226, 230);
+    --background-color: rgba(0, 0, 0, 0.05);
+    padding: 1.5rem 2rem;
+    border-radius: 6px;
+    border: var(--border-color) 1px solid;
+    width: fit-content;
+    margin: 2rem auto 0;
+}
+
+.feelingsLog table {
+    border-collapse: collapse;
```

Comment:
nice attention to detail here building the table

Reply:
Agreed, also using rems will make it much easier to make it all responsive!